python_modules/dagster/dagster/_core/definitions/job_definition.py

```diff
@@ -794,28 +794,6 @@ def _get_job_def_for_asset_selection(
                 " or job."
             )
 
-        # Test that selected asset checks can be run individually. Currently this is only supported
```

Comment:
without this, we fall back a slightly less interpretable error. But we'll be replacing that with a better error once I land atomic execution unit ids on checks